.github/actions/translation-tracker/README.md

```diff
@@ -0,0 +1,266 @@
+# p5.js Translation Tracker
+
+A comprehensive GitHub Action and command-line tool to track translation status across multiple languages in the p5.js website examples.
+
+## 🚀 Overview
+
+This tool helps maintain translation consistency by:
+- **Week 1**: Basic file-based change detection using Git
+- **Week 2**: GitHub API integration for accurate commit tracking and automated issue creation
+- **Week 3**: Multi-language support with single issues per file and manual scanning capabilities
+
+## 📋 Features
+
+### ✅ Week 1 Features (File-based tracking)
+- Git-based change detection for English example files
+- File modification time comparison
+- Support for all languages: Spanish (es), Hindi (hi), Korean (ko), Chinese Simplified (zh-Hans)
```

Comment:
Minor: a shorter README is easier to maintain, so please remove the duplicated info. (The supported languages are listed below already in a very readable format)